WMF Framework/NSRegularExpression+Utilities.swift

```diff
@@ -0,0 +1,35 @@
+public extension NSRegularExpression {
+    func firstMatch(in string: String) -> NSTextCheckingResult? {
+        return firstMatch(in: string, options: [], range: string.fullRange)
+    }
+    
+    func firstReplacementString(in string: String, template: String = "$1") -> String? {
+        guard let result = firstMatch(in: string)
+        else {
+            return nil
+        }
+        return replacementString(for: result, in: string, offset: 0, template: template)
+    }
+    
+    func stringByReplacingMatches(in string: String, options: NSRegularExpression.MatchingOptions = [], template: String? = nil, with block: (String) -> String) -> String {
```

Comment:
Nope - good catch. I wrote it for replacing `{{templateVars}}` in js but I ended up working around it in another way. There's still one instance in `PropertiesScript` but that's easy enough to do a search and replace because it's a single var

Reply:
Removed